extension/lib/main.js

```diff
@@ -113,14 +110,65 @@ exports.main = function (options, callbacks) {
         widget.tooltip = config.extension.widget_tooltips[data];
         break;
     }
+  });*/
+
+  var { Frame } = require("sdk/ui/frame");
+  var { ToggleButton } = require('sdk/ui/button/toggle');
+  var { Toolbar } = require("sdk/ui/toolbar");
+
+  var frame = new Frame({
+    url: "./frame/frame.html"
   });
 
+  var menuButton = ToggleButton({
+    id: "menu-button",
+    label: "Menu",
+    icon: {
+      "32" : "./addon32.png",
+      "64" : "./addon64.png"
+    },
+    onChange: handleMenuButtonChange
+  });
+
+  var logButton = ToggleButton({
+    id: "log-button",
+    label: "Logging",
+    icon: {
+      "32" : "./addon32.png",
+      "64" : "./addon64.png"
```

Comment:
Sadly SDK doesn't support text buttons. :(

I'm playing with a logo like this.

![log_file_format_64](https://cloud.githubusercontent.com/assets/568751/8235308/77a09542-15e1-11e5-9e21-13e3e147c4fa.png)

I like it. And you?